<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Conformity Window</string>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Activity Group&lt;/span&gt;&lt;/p&gt;&lt;p&gt;This Options let you choose how activities influences Conformity.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,7">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>142</width>
          <height>349</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Mode</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QRadioButton" name="m_rg_kmeans">
              <property name="toolTip">
               <string/>
              </property>
              <property name="text">
               <string>K-Means++</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="m_rg_random">
              <property name="toolTip">
               <string/>
              </property>
              <property name="text">
               <string>Random</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Algorithm</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QRadioButton" name="m_rb_alg_auto">
              <property name="text">
               <string>Auto</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="m_rb_alg_full">
              <property name="text">
               <string>Full</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="m_rb_alg_elkan">
              <property name="text">
               <string>Elkan</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="title">
            <string>Number of Clusters</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QSpinBox" name="m_sb_n_clusters">
              <property name="minimum">
               <number>2</number>
              </property>
              <property name="maximum">
               <number>9999999</number>
              </property>
              <property name="value">
               <number>5</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Max Iterations</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QSpinBox" name="m_sb_max_iter">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>100000</number>
              </property>
              <property name="singleStep">
               <number>50</number>
              </property>
              <property name="value">
               <number>300</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="10,0,1">
       <item>
        <widget class="QTextBrowser" name="m_textBrowser">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="autoFormatting">
          <set>QTextEdit::AutoAll</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="m_progressBar">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="m_run_algorithm_button">
           <property name="text">
            <string>Run Algorithm</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_exportResultButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Save Result Log</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_close_button">
           <property name="text">
            <string>Close</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
